explainer.md

```diff
@@ -0,0 +1,386 @@
+Provider) or have poor user privacy (e.g. data-collection or tracking the user
+for risk analysis, to provide a 'frictionless flow'). The payments industry
+needs a consistent, low friction, strong authentication flow for online
+payments.
+
+The [Web Authentication (WebAuthn) API][webauthn] makes FIDO-based
+authentication available on the web, which provides a strong, low-friction
+method for a user to prove a pre-established identity with a given site.
+However, WebAuthn is not immediately suitable to solve the payments
+authentication problem, as:
+
+1. It requires the Account Provider to be present in the transaction (e.g. via
+   an embedded iframe), which increases user friction and lowers conversion
+   rates.
+1. The generated assertion contains no payments-related information, so it
+   cannot be used as-is to fulfill regulatory requirements to provide evidence
+   of user content (e.g. [Dynamic Linking] requirements).
+1. WebAuthn does not allow credential creation in a cross-origin iframe, thus
+   excluding a useful onboarding flow - enrolling a Customer after they have
+   completed a traditional authentication flow (e.g. via SMS OTP), without
+   the friction of redirecting them to the bank's site or app.
+
+This proposal attempts to provide an authentication solution for online
+payments that is both as strong and low-friction as WebAuthn, whilst also
+solving these three issues.
+
+### Goals
+
+Find a solution that (in no particular order):
+
+* Has as-strong or stronger authentication than current challenge flows.
+* Is less friction for the Customer than current challenge flows.
+* Is more protective of user-privacy than current frictionless flows.
+* Improves the ability for online payments to meet regulations such as [Dynamic
+  Linking].
+
+> **NOTE**: It is out-of-scope for the specification to **prove** adherence
+> to e.g. Dynamic Linking, but we aim to produce a solution that could be
+> vetted as such.
+
+* Allows for in-flow enrollment of Customers during a traditional challenge
+  flow, as well as outside of a transaction.
+
+### Non-Goals
+
+* Selection of a Payment Instrument by the Customer; it is presumed that the
+  Customer has already done so (e.g. by typing in their credit card number).
+* ID & V to establish real world identity during enrollment; it is up to the
+  Account Provider to determine the Customer's identity to their satisfaction.
+* Providing authentication for peer-to-peer or business-to-business
+  transactions.
+    * We expect that Secure Payment Confirmation may be useful in these
+      cases, but are concentrating on consumer-to-business cases for now.
+
+## Proposed Solution: Secure Payment Confirmation
+
+Secure Payment Confirmation (SPC) builds on top of [WebAuthn] to add
+payment-specific datastructures to the signed assertion, and to relax
+assumptions to allow the API to be called in payment contexts.
+
+The intention for Secure Payment Confirmation is that a Customer would enroll
```

Comment:
I suggest "Ideally with Secure Payment Confirmation, a Customer would enroll..."

SPC can be used in other use cases (e.g., PSP-as-RP) and I think we intend to support those use cases. So I want the Account Provider use case to stand out as the "ideal" one from a scale perspective (which is the point I think you are making by saying "**any Merchant**".